tests/link/pytorch/test_extra_ops.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+import pytensor.tensor as pt
+from pytensor.configdefaults import config
+from pytensor.graph import FunctionGraph
+from pytensor.graph.op import get_test_value
+from tests.link.pytorch.test_basic import compare_pytorch_and_py
+
+
+def test_pytorch_CumOp():
+    """Test PyTorch conversion of the `CumOp` `Op`."""
+
+    # Create a symbolic input for the first input of `CumOp`
+    a = pt.matrix("a")
+
+    # Create test value tag for a
+    a.tag.test_value = np.arange(9, dtype=config.floatX).reshape((3, 3))
```

Comment:
No need for test values and tags. We're planning to deprecate that functionality as well